flink-connectors/flink-connector-kafka-0.10/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer010.java

```diff
@@ -130,7 +162,7 @@ public FlinkKafkaConsumer010(List<String> topics, DeserializationSchema<T> deser
 	 *           The properties that are used to configure both the fetcher and the offset handler.
 	 */
 	public FlinkKafkaConsumer010(List<String> topics, KafkaDeserializationSchema<T> deserializer, Properties props) {
-		super(topics, deserializer, props);
+		this(topics, null, deserializer, props);
```

Comment:
Same here with the `null` value for `Pattern`. I think it would be good to avoid it if possible.

Reply:
Will add Nullable with the same reasoning as for the list of topics.